packages/stream_chat_flutter/lib/src/message_list_view.dart

```diff
@@ -239,6 +239,8 @@ class _MessageListViewState extends State<MessageListView> {
 
   final MessageListController _messageListController = MessageListController();
 
+  Map<String, VideoPackage> videoPackages = {};
```

Comment:
This can be final

Reply:
It's in the state, I don't think this matters that much.